test/com/facebook/buck/features/apple/project/ProjectIntegrationTest.java

```diff
@@ -627,6 +627,19 @@ public void testHalide() throws IOException {
     workspace.verify();
   }
 
+  @Test
+  public void testBuckProjectGeneratedSchemeWithEnvVariablesAndExpandSetting() throws IOException {
+    ProjectWorkspace workspace =
+      TestDataHelper.createProjectWorkspaceForScenario(
+        this, "project_generated_scheme_with_env_variables_and_expand_setting", temporaryFolder);
+    workspace.setUp();
+
+    ProcessResult result = workspace.runBuckCommand("project", "//Apps:workspace");
+    result.assertSuccess();
+
+    workspace.verify();
+  }
```

Comment:
Should the the generated scheme be compared with TestApp.xcscheme.expected?

Reply:
`workspace.verify()` do this